client/js/views/Stats.vue

```diff
@@ -0,0 +1,151 @@
+      </h4>
+    </section>
+    <section class="px-8">
+      <!-- Season Champions -->
+      <div class="mb-10">
+        <h2
+          v-if="
+            selectedSeason &&
+              (selectedSeason.bracketLink ||
+                selectedSeason.firstPlace ||
+                selectedSeason.secondPlace ||
+                selectedSeason.thirdPlace)
+          "
+          class="text-h2 mb-4"
+        >
+          Season Champions
+        </h2>
+        <div class="d-flex justify-space-around flex-wrap">
+          <award-card
+            v-if="selectedSeason && selectedSeason.firstPlace"
+            :username="selectedSeason.firstPlace"
+            :place="1"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.secondPlace"
+            :username="selectedSeason.secondPlace"
+            :place="2"
+            class="mb-4"
+          />
+          <award-card
+            v-if="selectedSeason && selectedSeason.thirdPlace"
+            :username="selectedSeason.thirdPlace"
+            :place="3"
+            class="mb-4"
+          />
+        </div>
+        <p v-if="selectedSeason && selectedSeason.bracketLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.bracketLink" data-cy="tournament-bracket-link">
+            here to see the official tournament bracket
+          </a>
+        </p>
+        <p v-if="selectedSeason && selectedSeason.footageLink" class="text-body-1">
+          Click
+          <a :href="selectedSeason.footageLink" data-cy="tournament-footage-link">
+            here to watch the official tournament footage
+          </a>
+          with play-by-play commentary.
+        </p>
+      </div>
+      <!-- Rankings Table -->
+      <h2 class="text-h2 ml-4">
+        Weekly Rankings
+        <stats-scoring-dialog />
+      </h2>
+      <stats-leaderboard :loading="loadingData" :season="selectedSeason" />
+    </section>
+  </div>
+</template>
+
+<script>
+import { io } from '@/plugins/sails.js';
+import AwardCard from '@/components/AwardCard.vue';
+import StatsLeaderboard from '@/components/StatsLeaderboard.vue';
+import StatsScoringDialog from '@/components/StatsScoringDialog.vue';
+
+const dayjs = require('dayjs');
+
+export default {
+  name: 'Stats',
+  components: {
+    AwardCard,
+    StatsLeaderboard,
+    StatsScoringDialog,
+  },
+  data() {
+    return {
+      loadingData: false,
+      selectedSeason: null,
+      seasons: [],
+    };
+  },
+  computed: {
+    seasonStartString() {
+      if (this.selectedSeason) {
+        return dayjs(this.selectedSeason.startTime).format('YYYY/MM/DD');
+      }
+      return '';
+    },
+    seasonEndString() {
+      if (this.selectedSeason) {
+        return dayjs(this.selectedSeason.endTime).format('YYYY/MM/DD');
+      }
+      return '';
+    },
+  },
+  created() {
+    this.loadingData = true;
+    io.socket.get('/stats', (res, jwres) => {
```

Comment:
Do all API calls need to be via socket, or just game calls? It feels like you might be able to have these just be api GET requests to a net new endpoint specifically for data that could be cached more easily. Then again, I don't really know enough about sockets to say for sure.

Reply:
Currently all api calls are expected to come in via socket. This is worth revisiting holistically because there's no particular need for that to be the case, other than centralization of config (i.e. pointing api requests to the right domain both for sockets and for the other fetch approach)